MSBuild.Sdk.Extras/Sdk/Sdk.targets

```diff
@@ -0,0 +1,12 @@
+<Project>
+
+	<PropertyGroup>
+		<MSBuildAllProjects>$(MSBuildAllProjects);$(MSBuildThisFileFullPath)</MSBuildAllProjects>
+	</PropertyGroup>
+
+	<Import Sdk="Microsoft.NET.Sdk" Project="Sdk.targets" />
```

Comment:
Has the order of this import and the Extras targets import been tested when used in this way? As far as I've seen, the Extras targets have to be before the Sdk.Targets, which is why the readme tells people to add the conditional import within their project file. Using this as an Sdk is nicer, once Sdk's can be distributed as NuGets, but the order should probably be the same?

Reply:
I checked and built two sample projects (uwp, android) and It doesn't seem to affect at all.
I'm looking for corner cases by using more complicated project setup.